src/middlewared/middlewared/plugins/pool.py

```diff
@@ -1590,11 +1590,20 @@ def __find_disk_from_topology(self, label, pool):
             verrors.add('id', 'Pool already locked.')
 
         if not verrors:
-            if not await self.middleware.call('disk.geli_testkey', pool, passphrase):
+            # Let's also make sure that this pool is not being used by system dataset service
```

Comment:
Don't need the first part:
```suggestion
            # Make sure that this pool is not being used by system dataset service
```